Tests/JournalMonitorTests.cs

```diff
@@ -1108,5 +1106,14 @@ public void TestStatisticsWithoutThargoidEncounters()
             StatisticsEvent statisticsEvent = (StatisticsEvent)events[0];
             Assert.AreEqual(7083, statisticsEvent.multicrew.timetotalseconds);
         }
+
+        [TestMethod]
+        public void TestReputationWithoutIndependent()
+        {
+            string line = @"{ ""timestamp"":""2019 - 09 - 13T23: 08:20Z"", ""event"":""Reputation"", ""Empire"":18.287001, ""Federation"":75.703102, ""Alliance"":1.179020 }";
+            List<Event> events = JournalMonitor.ParseJournalEntry(line);
+            CommanderReputationEvent reputationEvent = (CommanderReputationEvent)events[0];
+            Assert.AreEqual(18.287001M, reputationEvent.empire);
```

Comment:
Add asserts for Federation and Alliance reps, comment on why we aren't testing Independent rep?

Reply:
I'll go one better and assert that the Independent rep is null.

Reply:
Why make our test fail if FDev fix Independent rep?

Reply:
It won't, that's a canned event.

Reply:
Ah, of course.